phpunit.xml.dist

```diff
@@ -41,13 +42,15 @@
         <whitelist>
             <directory>./src/Symfony/</directory>
             <exclude>
+                <directory>./src/Symfony/Abstractions/*/Tests</directory>
```

Comment:
Should this also be `Abstraction`? See PSR-4 configuration above.

Reply:
fixed thanks